packages/paste-website/src/pages/components/code-block/index.mdx

```diff
@@ -0,0 +1,234 @@
+| `element?` | `string` | Overrides the default element name to apply unique styles with the Customization Provider | `'CODE_BLOCK_WRAPPER'` |
+
+##### CodeBlockHeader
+
+| Prop       | Type                                                         | Description                                                                               | Default               |
+| ---------- | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------- | --------------------- |
+| `children` | `string`                                                     | The code snippet to be rendered.                                                          |                       |
+| `element?` | `string`                                                     | Overrides the default element name to apply unique styles with the Customization Provider | `'CODE_BLOCK_HEADER'` |
+| `as?`      | `'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div', 'label', 'span'` | The HTML element to render                                                                | `'h3'`                |
+
+##### CodeBlockTabList
+
+| Prop       | Type     | Description                                                                               | Default                 |
+| ---------- | -------- | ----------------------------------------------------------------------------------------- | ----------------------- |
+| `children` | `string` | The code snippet to be rendered.                                                          |                         |
+| `element?` | `string` | Overrides the default element name to apply unique styles with the Customization Provider | `'CODE_BLOCK_TAB_LIST'` |
+
+##### CodeBlockTab
+
+| Prop       | Type     | Description                                                                               | Default            |
+| ---------- | -------- | ----------------------------------------------------------------------------------------- | ------------------ |
+| `children` | `string` | The code snippet to be rendered.                                                          |                    |
+| `element?` | `string` | Overrides the default element name to apply unique styles with the Customization Provider | `'CODE_BLOCK_TAB'` |
+
+##### CodeBlockTabPanel
+
+| Prop       | Type     | Description                                                                               | Default                  |
+| ---------- | -------- | ----------------------------------------------------------------------------------------- | ------------------------ |
+| `children` | `string` | The code snippet to be rendered.                                                          |                          |
+| `element?` | `string` | Overrides the default element name to apply unique styles with the Customization Provider | `'CODE_BLOCK_TAB_PANEL'` |
+
+##### CodeBlock
```

Comment:
suggestion: I think this prop table should go first or second, since it's probably the one people will be referring to most.

Reply:
Good point!